### ISSUE TITLE
layer names are too long

### ISSUE THREAD

**Issue by gon9031** [mlperf/mobile_app#92]
I found that the layer names of these models are very long.

deeplabv3_mnv2_ade20k_float.tflite
ssd_mobilenet_v2_300_float.tflite

The names are as like below:
e.g. "FeatureExtractor/MobilenetV2/expanded_conv/depthwise/BatchNorm/FusedBatchNormV3;FeatureExtractor/MobilenetV2/expanded_conv/depthwise/depthwise;FeatureExtractor/MobilenetV2/expanded_conv_5/project/BatchNorm/FusedBatchNormV3;FeatureExtractor/MobilenetV2/expanded_conv_5/project/weights;FeatureExtractor/MobilenetV2/expanded_conv_5/project/Conv2D_1"

Due to too long names, potential compilation parsing errors can be caused.
Please change the name normally.

**Comment by thaink** [comment]
Hi Terry,
How do you think about this issue?

**Comment by freedomtan** [comment]
It seems this is what MLIR-based converter uses to keep names unique.

**Comment by terryheo** [comment]
@thaink it's a feature request to TFLite converter. So I think it should be routed to https://github.com/tensorflow/tensorflow/issues

**Comment by freedomtan** [comment]
Quick solution, assuming the one with long names is `/tmp/foo.tflite`, you can get tflite with much shorter names with 
```
$ bazel build --config opt //tensorflow/compiler/mlir/lite:flatbuffer_translate
$ bazel-bin/tensorflow/compiler/mlir/lite/flatbuffer_translate --tflite-flatbuffer-to-mlir /tmp/foo.tflite | bazel-bin/tensorflow/compiler/mlir/lite/flatbuffer_translate --mlir-to-tflite-flatbuffer > /tmp/bar.tflite
```
